pkg/virt-controller/services/virtiofs.go

```diff
@@ -28,7 +28,7 @@ func generateVirtioFSContainers(vmi *v1.VirtualMachineInstance, image string, co
 	for _, volume := range vmi.Spec.Volumes {
 		if _, isPassthroughFSVolume := passthroughFSVolumes[volume.Name]; isPassthroughFSVolume {
 
-			container := generateContainerFromVolume(&volume, image, config)
+			container := generateContainerFromVolume(&volume, image, config, vmi.IsCPUDedicated(), *vmi.Status.QOSClass == k8sv1.PodQOSGuaranteed)
```

Comment:
I think boolean arguments should be avoided as much as possible.

In this particular case, it is easy: you can just refactor the existing `generateContainerFromVolume` to receive the list of resources - then the callers would generate the list of required resources for the container, and pass it along the function.

Reply:
understook. working on it